atomic_reactor/inner.py

```diff
@@ -9,7 +9,7 @@
 Script for building docker image. This is expected to run inside container.
 """
 
-from __future__ import absolute_import
+from __future__ import absolute_import, division
```

Comment:
In this module I see we are working with storage sizes:

```
mb_free=st.f_bfree * st.f_frsize / mb,
```
Shouldn't be this rewrited to

```python
mb_free=st.f_bfree * st.f_frsize // mb,
```
To keep integers instead of floats?

Reply:
Please check required types on all places where division is used

Reply:
Yes, for mb_free etc we want integers (i.e. same as we had before).